libraries/src/User/UserFactory.php

```diff
@@ -75,4 +75,4 @@ public function loadUserByUsername(string $username): User
 
         return $this->loadUserById((int) $this->db->loadResult());
     }
-}
+}
```

Comment:
```suggestion
}

```
Revert end of file change.

Reply:
This and the other one will also fix PHPCS failing.